.gitignore

```diff
@@ -1,6 +1,5 @@
 # node / npm stuff
 /node_modules
-/package-lock.json
```

Comment:
We have internal discussions several times about having package-lock.json in this git repo and our current consensus is not to have it to avoid conflicts among pull requests. We know that there are different opinions on this but can you revert this change and the package-lock.json addition?

Reply:
No worries. Personally I think it would be useful because I think the ^0.26.1 is what caused the issue in the first place, but I understand the reasoning